src/main/java/com/github/ciomarabanu/aoc/_2021/Day10.java

```diff
@@ -0,0 +1,43 @@
+package com.github.ciomarabanu.aoc._2021;
+
+import com.github.ciomarabanu.aoc.utils.FileReaderFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Stack;
+
+public class Day10 {
+    public static void main(String[] args) {
+        var reader = FileReaderFactory.build();
+        var paranthesis = reader.readFileAsStrings("2021/day10");
+        System.out.println(getParenScore(paranthesis));
+    }
+
+    public static int getParenScore(List<String> paren) {
```

Comment:
I think part 1 is asking for something like syntax error score so what do you think about renaming this method to `getSyntaxErrorScore`?